ios/Swift/imagepicker/ImagePickerViewController.swift

```diff
@@ -64,78 +64,84 @@ extension ViewController {
             
             
             // Use SwiftyJSON to parse results
-            let json = JSON(data: dataToParse)
-            let errorObj: JSON = json["error"]
-            
-            self.spinner.stopAnimating()
-            self.imageView.isHidden = true
-            self.labelResults.isHidden = false
-            self.faceResults.isHidden = false
-            self.faceResults.text = ""
-            
-            // Check for errors
-            if (errorObj.dictionaryValue != [:]) {
-                self.labelResults.text = "Error code \(errorObj["code"]): \(errorObj["message"])"
-            } else {
-                // Parse the response
-                print(json)
-                let responses: JSON = json["responses"][0]
+            do {
+                let json = try JSON(data: dataToParse)
```

Comment:
This try is unneeded. You can just run as:

    let json =  JSON(data: dataToParse)